roles/openshift_master/tasks/main.yml

```diff
@@ -86,7 +86,7 @@
   when: not openshift.common.is_containerized | bool
 
 - name: Get docker images
-  command: docker images
+  shell: docker images | awk '{  print $1 "\:" $2 }'
```

Comment:
I think it would be better to have create a filter to parse the output of docker images rather than using shell/awk.